### PR TITLE
Add union, intercept, intersect abilities of queries

### PR DESCRIPTION
**Pull request by yawen-QvQ**

### CODE REVIEW

File: src/main/scala/io/flow/postgresql/CombinedQuery.scala [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+) extends CombinedQuery {
+
+  override val operation = "except"
+
+  override def bind[T](name: String, value: T): SQLBase = {
+    this.copy(
+      left = left.bind(name, value),
+      right = right.bind(name, value)
+    )
+  }
```
Comment (benwaffle) [inline src/main/scala/io/flow/postgresql/CombinedQuery.scala:79]: We can put this bind implementation in CombinedQuery

File: src/main/scala/io/flow/postgresql/CombinedQuery.scala [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+  override def withDebugging(): SQLBase = this.copy(debug = true)
```
Comment (benwaffle) [inline src/main/scala/io/flow/postgresql/CombinedQuery.scala:81]: as well as this

File: src/test/scala/io/flow/postgresql/CombinedQuerySpec.scala [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+        " - id: 6",
+        "Interpolated:",
+        "select * from users where id = 6)"
+      ).mkString("\n")
+    )
+  }
+
+  it("binding on query") {
+    val query1 = Query("select id from users").and("email = {email}").bind("email", Some("foo@flow.io"))
+    val query2 = Query("select id from users").and("email = {email}").bind("email", Some("bar@flow.io"))
```
Comment (benwaffle) [inline src/test/scala/io/flow/postgresql/CombinedQuerySpec.scala:90]: I'm pretty sure you can't bind the same variable twice. I would have expected this to fail at `.anormSql()`

**Comment by benwaffle** [comment]
@mbryzek PR is not fully completed, but looking for a review of the design here.

**Review by mbryzek — COMMENTED** [review]
I'm bearish on this change - the interface here doesn't really lend itself well to this type of composition, and as it comes up rarely at flow, suggest we hold on this change for now (falling back to using anorm sql directly as needed)

**Comment by mbryzek** [comment]
closing due to age

**Pull request closed without merging**